src/org/traccar/Main.java

```diff
@@ -35,6 +36,20 @@ public static void main(String[] args) throws Exception {
             Context.getWebServer().start();
         }
 
+        //added by Erez
+        Timer timer = new Timer();
+        timer.scheduleAtFixedRate(new TimerTask() {
+            @Override
+            public void run() {
+                // Clean positions history every day
+                try {
+                    Context.getDataManager().clearPositionsHistory();
+                } catch (Exception error) {
+                    Log.warning(error);
+                }
+            }
+        }, 10*1000, 24*60*60*1000);
```

Comment:
Move constants to static variables at the top of the class and give them some name.